team-config.yml

```diff
@@ -94,3 +94,12 @@ help-with-fees:
     build_notices_channel: "#pet-devops"
   tags:
     application: help-with-fees
+Child Arrangement Service:
```

Comment:
this should be the 'product' not display name
```suggestion
c100:
```

Reply:
Thanks, Tim. Changed it to product name.